### DISCUSSION TITLE
Issue using openAPI for updating/deleting table rows

### DISCUSSION THREAD

**Discussion by smitsialis** [Budibase/budibase#6832] [Help] [answered]
I'm trying to use the OpenAPI to manage the data in my tables.  I'm using Postman to test this and having issues with the delete row and update row.  I am able to create new rows, search for rows and retrieve a row using the id.
Example:
`GET {{baseUrl}}/tables/datasource_plus_8bf0184b1acd4f6a8940ae60eebbc1a4__Contracts/rows/7 `
 Works fine:
```
 {
    "data": {
        "id": 7,
        "Name": "contract_YVES5",
        "_id": "%5B7%5D",
        "tableId": "datasource_plus_8bf0184b1acd4f6a8940ae60eebbc1a4__Contracts"
    }
}
```
I am trying to edit this using:
```
PUT {{baseUrl}}/tables/datasource_plus_8bf0184b1acd4f6a8940ae60eebbc1a4__Contracts/rows/7 
{
  "_id": "%5B6%5D",
  "type": "row",
  "tableId": "datasource_plus_8bf0184b1acd4f6a8940ae60eebbc1a4__Contracts",
  "Name": "New_data"
}
```
but get back:
```
{
    "message": "missing",
    "status": 404
}
```

Same error when I try to delete:
`DELETE {{baseUrl}}/tables/datasource_plus_8bf0184b1acd4f6a8940ae60eebbc1a4__Contracts/rows/7`

The headers are the same for all 3 methods and not the problem.  The table has a single column 'Name'.  What I'm I missing?

**Comment by melohagan** [comment, marked as answer]
Hi @smitsialis 

Thanks for posting - this appears to be a bug. I've raised the issue here: https://github.com/Budibase/budibase/issues/6880
